### ISSUE TITLE
Error in IDD for ZoneHvac:CoolingPanel:RadiantConvective:Water

### ISSUE THREAD

**Issue by Lefort-Antoine** [NREL/EnergyPlus#7537]
The IDD has this:
```
ZoneHVAC:Baseboard:RadiantConvective:Electric,
       \extensible:2 - repeat last two fields, remembering to remove ; from "inner" fields.
       \memo The number of surfaces can be expanded beyond 100, if necessary, by adding more
       \memo groups to the end of the list
       \min-fields 18
  A1 , \field Name
       \required-field
       \reference ZoneEquipmentNames
```
should be:
```
ZoneHVAC:Baseboard:RadiantConvective:Electric,
       \extensible:2 - repeat last two fields, remembering to remove ; from "inner" fields.
       \memo The number of surfaces can be expanded beyond 100, if necessary, by adding more
       \memo groups to the end of the list
       \min-fields 18
  A1 , \field Name
       \required-field
       \reference-class-name validBranchEquipmentTypes
       \reference validBranchEquipmentNames
       \reference ZoneEquipmentNames
```

**Comment by jmarrec** [comment]
@Lefort-Antoine You are showing the IDD for the Electric kind, which logically shouldn't be a valid branch equipment.
The Water kind is fine:

```
ZoneHVAC:Baseboard:RadiantConvective:Water,
       \extensible:2 - repeat last two fields, remembering to remove ; from "inner" fields.
       \memo The number of surfaces can be expanded beyond 100, if necessary, by adding more
       \memo groups to the end of the list
       \min-fields 12
  A1,  \field Name
       \required-field
       \reference-class-name validBranchEquipmentTypes
       \reference validBranchEquipmentNames
       \reference ZoneEquipmentNames
```

**Comment by Lefort-Antoine** [comment]
Sorry. As stated the issue title I wanted to refer to

```
ZoneHVAC:CoolingPanel:RadiantConvective:Water,
       \extensible:2 - repeat last two fields, remembering to remove ; from "inner" fields.
       \memo The number of surfaces can be expanded beyond 100, if necessary, by adding more
       \memo groups to the end of the list
       \min-fields 18
  A1 , \field Name
       \required-field
       \reference ZoneEquipmentNames 
```
It should be
```
ZoneHVAC:CoolingPanel:RadiantConvective:Water,
       \extensible:2 - repeat last two fields, remembering to remove ; from "inner" fields.
       \memo The number of surfaces can be expanded beyond 100, if necessary, by adding more
       \memo groups to the end of the list
       \min-fields 18
  A1 , \field Name
       \required-field
       \reference-class-name validBranchEquipmentTypes
       \reference validBranchEquipmentNames
       \reference ZoneEquipmentNames 
```

**Comment by RKStrand** [comment]
@Lefort-Antoine I just verified in the current version of develop as well as the recently released V9.2 that this fix has already been completed.  So, I think this is already done.  Could you verify that also and "comment and close" if you agree that this is already fixed?  Thanks!

**Comment by RKStrand** [comment]
@Lefort-Antoine Deleted a previous comment that was a mistake on my part.  Hope to fix this sometime this coming week.

**Comment by RKStrand** [comment]
Also addressing two minor doc errors from #7600 in this work.

**Comment by Myoldmopar** [comment]
Closed via #7607
